mobile/lib/screens/settings/language_switch.dart

```diff
@@ -66,7 +65,7 @@ class LanguageListState extends State<LanguageList> {
               Locale locale = await setLocale(language.languageCode);
               await AirQoApp.setLocale(context, locale);
               Navigator.pop(context, true);
-              //await Restart.restartApp();
+              await Restart.restartApp();
```

Comment:
Hi @katendejericho5 , please share more notes as to why we are reintroducing this App restart? Can a user "skip" it if they want to? cc: @BenjaminSsempala

Reply:
I noted when we just switch to another language say French, the KYA and Quiz remained in the former language, and this I think is caused due to the fact that Quiz and KYA are fetched when the app starts, so that's why we need to restart the app,

Reply:
I noted when we just switch to another language say French, the KYA and Quiz remained in the former language, and this I think is caused due to the fact that Quiz and KYA are fetched when the app starts, so that's why we need to restart the app